app/registries/foreman/settings/facts.rb

```diff
@@ -36,6 +36,7 @@
       'vlinuxbr*',
       'vovsbr*',
       'br-int',
+      'vif*,',
```

Comment:
As already pointed out by `jjvcuyler` in the thread:
```suggestion
      'vif*',
```

Reply:
Yeah typo, fixed.